common/anomalies/hfe_anomaly.txt

```diff
@@ -213,33 +215,29 @@ hfe_anomaly_blight = {
 	should_ai_and_humans_use = yes
 	null_spawn_chance = @null_spawn_chance
 	spawn_chance = {
+		base = 0
 		modifier = {
 			add = 1
 			host_has_dlc = "Synthetic Dawn Story Pack"
 			hfe_trigger_story_enabled = yes
-			hfe_trigger_hardcore_enabled = yes
+			OR = {
+				hfe_trigger_hardcore_enabled = yes
+				from.owner = { is_ai = yes }
+			}
 			uninhabitable_regular_planet = yes
 			exists = from.owner
 			from.owner = {
 				is_gestalt = no
 				num_owned_planets > 7
 			}
 		}
-		modifier = {
-			factor = 0.1
-			exists = from.owner
-			from.owner = {
-				is_ai = yes
-			}
-		}
-		modifier = {
-			factor = 2
-			years_passed > 30
-		}
-		modifier = {
-			factor = 2
-			years_passed > 60
-		}
+#		modifier = {
```

Comment:
why you commented it out

Reply:
I wanted the increasing chances to not include that specific period of time.

Reply:
I don't understand what you are trying to say. The AI and the player have the same chances of getting Blighted, at a one to one ratio. I removed the AI penalty in order to honor your request.

Reply:
AI have x10 more chance because you will have 10 AIs in game.

Reply:
Okay, done

Reply:
Okay, done